sormas-api/src/main/java/de/symeda/sormas/api/report/AggregateReportFacade.java

```diff
@@ -23,7 +23,11 @@ public interface AggregateReportFacade {
 
 	List<AggregateReportDto> getList(AggregateReportCriteria criteria);
 
+	List<AggregateReportDto> getAllAggregatedReportsFromSameEpiWeekUserAndJurisdiction(AggregateReportDto aggregateReportDto);
```

Comment:
```suggestion
	List<AggregateReportDto> getSimilarAggregateReports(AggregateReportDto aggregateReportDto);
```

The current name is too long in my eyes, I'd rather like you to add Javadoc to explain what the method uses to compare the aggregate reports.

Reply:
changed